File: pgm/ABC_R/ABC_319_A.r
con <- file(description = "stdin", open = "r")
arr <- readLines(con = con)
close(con)
S <- arr[1]
legendary_players <- new.env()
legendary_players[["tourist"]] <- 3858
legendary_players[["ksun48"]] <- 3679
legendary_players[["Benq"]] <- 3658
legendary_players[["Um_nik"]] <- 3648
legendary_players[["apiad"]] <- 3638
legendary_players[["Stonefeang"]] <- 3630
legendary_players[["ecnerwala"]] <- 3613
legendary_players[["mnbvmar"]] <- 3555
legendary_players[["newbiedmy"]] <- 3516
legendary_players[["semiexp"]] <- 3481
legendary_players[[S]] |>
    cat("\n", sep = "")
q("no")

File: pgm/ABC_Dart/ABC_319_A.dart
// import 'dart:collection';
import 'dart:io';
// import 'dart:math';
// import 'dart:typed_data';

void main() {
  final String S = stdin.readLineSync()!;
  const Map<String, int> m = {
    "tourist": 3858,
    "ksun48": 3679,
    "Benq": 3658,
    "Um_nik": 3648,
    "apiad": 3638,
    "Stonefeang": 3630,
    "ecnerwala": 3613,
    "mnbvmar": 3555,
    "newbiedmy": 3516,
    "semiexp": 3481,
  };
  print(m[S]);
  exit(0);
}
